Update/koto_005.txt

```diff
@@ -1514,7 +1514,7 @@ void main()
 
 //rとりあえず、それは彼の人柄と普段の言動からある程度の納得でもって受け入れられているようだが、さすがにあまり人前に出ないのは、変な勘繰りを生みかねない。
 	OutputLine(NULL, "とりあえず、それは彼の人柄と普段の言動からある程度の納得でもって受け入れられているようだが、さすがにあまり人前に出ないのは、変な勘繰りを生みかねない。",
-			NULL, "The villagers appeared to have accepted that story to a certain extent based on Riku's established character and demeanor, but as expected, it was easy for strange misunderstandings to bloom since I never showed my face.", Line_Normal);
+			NULL, "The villagers appeared to have accepted that story to a certain extent based on Riku's established character and demeanor, but as expected, it was easy for undue suspicions to bloom since I never showed my face.", Line_Normal);
```

Comment:
```suggestion
			NULL, "The villagers appeared to have accepted that story to a certain extent based on Riku's established character and demeanor, but as expected, it was easy for undue suspicion to bloom since I never showed my face.", Line_Normal);
```